packages/react-app/src/components/NetworkSelector.jsx

```diff
@@ -97,30 +85,49 @@ export const NetworkSelector = props => {
   }, [setNetwork, setDefaultToken]);
 
   const onChange = network => {
+    close();
     setDefaultToken(network.value);
     setNetwork(network);
     setLocalNetwork(network.key);
     window.localStorage.setItem('chosenNetwork', network.key);
   };
 
+  const currentNetwork = networkOptions[localNetwork];
+  const selectOptions = networkOptions
+    .filter(network => currentNetwork.value !== network.value)
+    .map(network => {
+      return (
+        <SelectOption
+          onChange={onChange}
+          network={network}
+          key={network.key.toString()}
+        />
+      );
+    });
+
+  const handleOpen = isOpen ? close : open;
+
   return (
     <Flex {...props}>
-      <Select
-        onChange={onChange}
-        styles={customStyles}
-        fontWeight="bold"
-        defaultValue={networkOptions[localNetwork]}
-        value={networkOptions[localNetwork]}
-        options={networkOptions}
-        isClearable={false}
-        isSearchable={false}
-        hideSelectedOptions
-        components={{
-          Option: CustomSelectOption,
-          SingleValue: CustomSelectValue,
-          DropdownIndicator: CustomDropdownIndicator,
-        }}
-      />
+      <Popover isOpen={isOpen} placement="bottom">
+        <PopoverTrigger>
+          <Button
+            p={0}
+            _hover={{ background: 'transparent' }}
+            onClick={handleOpen}
+          >
+            <SelectValue {...currentNetwork} />
+          </Button>
+        </PopoverTrigger>
+        <PopoverContent
+          zIndex={4}
+          width="min-content"
+          border="1px solid rgba(226,232,240, 0.8)"
+          boxShadow="0 0.5rem 1rem #CADAEF"
```

Comment:
took this boxShadow from the recent [changes for responsive updates](https://github.com/raid-guild/xdai-omnibridge/pull/63/files#diff-423052922d3c6c95281b762e276349c4L70)